manifest.json

```diff
@@ -11,8 +11,7 @@
     "128": "icon128.png"
   },
   "options_ui": {
-      "page": "options.html",
-      "chrome_style": true
```

Comment:
Yep, you're right! Apparently, Firefox defaults to true for `browser_style` but I just updated the build script to explicitly declare it.

Related: [documentation](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/options_ui).